.github/CODEOWNERS

```diff
@@ -1,3 +1,17 @@
 # These owners will be the default owners for everything in
 # the repo, unless a later match takes precedence
-*       @hashicorp/cloud-experiences-go
\ No newline at end of file
+
+* @hashicorp/cloud-experiences-go
+
+# The Consul clients are owned by Consul
+/clients/cloud-consul-service/ @hashicorp/cloud-experiences-go @hashicorp/consul-cloud-ui @hashicorp/consul-cloud
+
+# The Vault clients are owned by Vault
+/clients/cloud-vault-service/ @hashicorp/cloud-experiences-go @hashicorp/vault-cloud-ui @hashicorp/vault-cloud
+
+# The Packer clients are owned by Packer
+/clients/cloud-packer-service/ @hashicorp/cloud-experiences-go @hashicorp/packer-cloud-ui @hashicorp/packer-cloud
+
+# The Resource Manager and Operation clients are owned by Dataplane
```

Comment:
I might be totally out of the loop on this, but I thought resource-manager (and maybe operation) were control plane

Reply:
yep you're right, double-checked cloud catalog, resource-manager is controlplane. operation is dataplane.